### PR TITLE
Changelog for 3.14.0

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,56 +2,50 @@
...
+	- Fixed standalone_self_upgrade not triggering because of stale data
 	  (ENT-4317)
 	- Fixed maintenance policy for promise log cleanup to respect history_length_days
 	  (ENT-4588)
-	- Improve efficiency and error handling of user specified policy update bundle
+	- Improved efficiency and error handling of user specified policy update bundle
 	- Log version of Enterprise agent outside of state (ENT-4352)
-	- Package module for managing windows packages using msiexec (ENT-3719)
-	- Prevent inventorying un-expanded free memory from cf-monitord
-	- Prevent mon.value_mem_total from being inventoried if not defined
```
Comment (roschern) [inline CHANGELOG.md]: Why remove mention of this small improvement?

Comment (olehermanse) [reply CHANGELOG.md]: I tried combining the 2 lines into 1 line which makes sense to me (pretending to be a normal user):

```
Prevented inventorying un-expanded memory values from cf-monitord (ENT-4522)
```

(Below)

Comment (roschern) [reply CHANGELOG.md]: makes sense.

**Pull request merged**